### PR TITLE
TASK] Add PHP Zip to container to use ZipArchive

### PR DESCRIPTION
**Pull request by thea7y**
b @visay

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -36,6 +36,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     php7.0-sqlite3 \
     php7.0-imagick \
     php7.0-redis \
+    php7.0-zip \
```
Comment (Nguonchhay) [inline Dockerfile]: @thea7y I think you should move this line to below  `php7.0-xdebug \`

**Comment by visay** [comment]
Please update the CHANGELOG

**Comment by Nguonchhay** [comment]
@visay it will be merged to php7 branch. How to update the CHANGELOG?

**Comment by visay** [comment]
Edit this file https://github.com/Sebobo/Shel.DockerFlow/blob/master/CHANGELOG

Send merge request to master branch and then rebase php7 branch

**Comment by thea7y** [comment]
b @visay, I made changes. Please review and merge accordingly.

**Pull request merged**